Makefile

```diff
@@ -23,12 +23,16 @@ tests:
 	python3 testsuite/gcmc_tests.py
 	python3 testsuite/weak_polyelectrolyte_dialysis_test.py
 	python3 testsuite/globular_protein_tests.py
+
 sample:
-	python3 sample_scripts/peptide_simulation_example.py
+	python3 samples/peptide.py
 
 visual:
-	python3 handy_scripts/vmd-traj.py
```

Comment:
`make visual`  and `make sample` don't work.

Reply:
thank you for solving this bug!

Reply:
@davidbbeyer can you please check these two Makefile targets? They still break on my workstation after applying the bug fixes.

Reply:
@jngrad @davidbbeyer  I solved the issue in `samples\peptide.py`, it was actually broken since #24. `make visual` works well in my work station...

Reply:
Here is what I get:
```
$ make visual
python3 visualization/vmd-traj.py
Loading frame ./frames/trajectory0.vtf
...
Loading frame ./frames/trajectory105.vtf
Traceback (most recent call last):
  File "/work/jgrad/pyMBE/visualization/vmd-traj.py", line 189, in <module>
    at_type=int(at_list[-1])
ValueError: invalid literal for int() with base 10: 'nH'
```

Reply:
Looks to me that is trying to load some broken trajectory file, how did you produce those frames? Did the simulation finish or did you stop it before finishing it?

Reply:
Ouch, I didn't realize it was reading frames created by `samples/peptide.py` :smiley: I erased those files and regenerated them by running the fixed `samples/peptide.py` script, and now it works.